GitUI/CommandsDialogs/FormDeleteRemoteBranch.cs

```diff
@@ -64,14 +65,36 @@ private void RecalculateSizeConstraints()
             ResumeLayout();
         }
 
+        private List<IGitRef> GetSelectedRemotRefs() => Branches.GetSelectedBranches().ToList();
+
+        private void Branches_SelectedValueChanged(object? sender, EventArgs e)
```

Comment:
Please keep events handlers after instance methods. Bonus points for keeping those in alphabetical order.